checks/evaluation/pinned_dependencies.go

```diff
@@ -80,7 +106,7 @@ func PinningDependencies(name string, c *checker.CheckRequest,
 				Text:      *rr.Msg,
 				Snippet:   rr.Location.Snippet,
 			})
```

Comment:
We probably need a `continue` if `rr.Msg != nil` because otherwise the errored Dependency would count as being unpinned (if I'm reading this right) ?

Reply:
Not sure, but I will try to explain. At this point, we have already classified all dependencies as pinned or unpinned and we are evaluating the dependencies. The previous behaviour was "if dependency has error, then log debug, otherwise IT IS an unpinned dependency, then log warn". The current behavior is "if dependency has error, then log debug, otherwise IF IT IS an unpinned dependency, then log warn". Does that make sense?